committees/models.py

```diff
@@ -71,7 +72,8 @@ class AssociationGroupMembership(models.Model):
     For another we want additional information stored that is difficult when attempting to override the link between
     user and group.
     """
-    member = models.ForeignKey(Member, on_delete=models.CASCADE)
+    member = models.ForeignKey(Member, on_delete=models.CASCADE, blank=True, null=True)
+    external_person = models.CharField(max_length=64, help_text="Person not is not a member", blank=True, null=True)
```

Comment:
No need for `null=True` on a non-unique Charfield.
(there's also a double "not" in the help text)